testing/contract/examples/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -x
+set -e
+
```

Comment:
```suggestion
# Fail if build_dir is not set
${build_dir:?}
```

How about something like this?

Reply:
Shell check says it's not set, so that will fail, right?

Reply:
It is one of the suggested fixes for the warning, https://www.shellcheck.net/wiki/SC2154. It fixes the warning in my machine